automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookClusterMonitoringSpec.scala

```diff
@@ -68,10 +67,10 @@ class NotebookClusterMonitoringSpec extends GPAllocFixtureSpec with ParallelTest
       implicit val ronToken: AuthToken = ronAuthToken
 
       val initialMachineConfig =
-        RuntimeConfig.DataprocConfig(
-          numberOfWorkers = 2,
-          masterMachineType = MachineTypeName("n1-standard-2"),
-          masterDiskSize = DiskSize(50),
```

Comment:
What's the reasoning for no longer using these types (`MachineTypeName` and `DiskSize`)?

Reply:
I think this is related to the changes we made to RuntimeConfig to RuntimeConfigRequest. But I have noticed that it is resulting in some failed tests due to the get cluster request still returning a ClusterCopy. This might be something we want to completely refactor in the near future